x/concentrated-liquidity/types/pool.go

```diff
@@ -0,0 +1,39 @@
+package types
+
+import (
+	proto "github.com/gogo/protobuf/proto"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+)
+
+type PoolI interface {
```

Comment:
Why is this called `PoolI`?

Let's call this `ConcentratedPoolExtension` please according to the spec and component diagram we discussed earlier

Reply:
Changed!